### PR TITLE
IGAPP-577: Remove feedback type selection

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/modules/app/components/App.js [hunk 1]
```diff
@@ -56,26 +58,28 @@ const App = () => {
...
-                    cityCode={cityCode}
-                  />
-                </NavigationContainer>
-              </IOSSafeAreaView>
-              {routeName && <PermissionSnackbarContainer routeName={routeName} />}
-            </>
-          </SafeAreaProvider>
-        </I18nProvider>
-      </StaticServerProvider>
+      <ThemeProvider theme={buildConfig().lightTheme}>
```
Comment (klinzo) [inline native/src/modules/app/components/App.js:61]: This allows us to use hooks to get the theme and write StyledComponents without having to pass the theme manually, like it is in web already.

File: native/src/routes/search/components/SearchFeedbackBox.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 // @flow
```
Comment (klinzo) [inline native/src/routes/search/components/SearchFeedbackBox.js:2]: Only converted to functional component

File: native/src/routes/search/components/NothingFoundFeedbackBox.js [hunk 3]
```diff
@@ -1,16 +1,16 @@
 // @flow
```
Comment (klinzo) [inline native/src/routes/search/components/NothingFoundFeedbackBox.js:1]: Only converted to functional component

File: release-notes/unreleased/IGAPP-577-remove-feedback-type-selection.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-577
+show_in_stores: true
+platforms:
+  - android
+  - ios
+en: To make it easier for you to send feedback we now automatically select the type of feedback for you!
```
Comment (Taggotty) [inline release-notes/unreleased/IGAPP-577-remove-feedback-type-selection.yml:6]:
```suggestion
en: To make it easier to send feedback we now automatically select the type of feedback for you!
```

**Review by Taggotty — APPROVED** [review]

**Review by SteffiStuffel — APPROVED** [review]
reviews, but not tested, because test-cms is down...

**Review by maxammann — APPROVED** [review]

**Pull request merged**